astropy/utils/masked/tests/test_function_helpers.py

```diff
@@ -157,6 +157,12 @@ def test_broadcast_to(self):
     def test_broadcast_arrays(self):
         self.check2(np.broadcast_arrays)
         self.check2(np.broadcast_arrays, subok=False)
+        # Regression test for bug for single array
+        ba = np.broadcast_arrays(self.ma, subok=True)
+        assert len(ba) == 1
```

Comment:
I was initially confused by this first new assertion. I think it'd help readability to also assert the return value is indeed a tuple.
```suggestion
        assert isinstance(ba, tuple)
        assert len(ba) == 1
```